File: poo/cplus2/struct_sec/area.cc
#include <iostream>

using namespace std;

class Area {
private:
    double base;
    double altura;

public:
    void leerDatos() {
        cout << "Ingrese la base del triangulo: ";
        cin >> base;
        cout << "Ingrese la altura del triangulo: ";
        cin >> altura;
    }

    double calcArea() {
        return (base * altura) / 2.0;
    }

    void mostArea() {
        cout << "El area del triangulo es: " << calcArea() << endl;
    }
};

int main() {
    Area t;
    t.leerDatos();
    t.mostArea();
    return 0;
}

File: poo/java/struct_sec/Area.java
import java.util.Scanner;

public class Area {
    private double base;
    private double altura;

    public void pedirDatos(Scanner leer) {
        System.out.print("Ingrese la base del triangulo: ");
        base = leer.nextDouble();
        System.out.print("Ingrese la altura del triangulo: ");
        altura = leer.nextDouble();
    }

    public double calcularArea() {
        return (base * altura) / 2.0;
    }

    public void mostArea() {
        System.out.println("El area del triangulo es: " + calcularArea());
    }
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        
        Area t = new Area();
        t.pedirDatos(leer);
        t.mostArea();
        
    }
}
